README.md

```diff
@@ -42,6 +42,7 @@ I renamed the repository to "Awesome Alternatives in Rust". The original name wa
 #### bash/PowerShell/fish
 
 * [nushell](https://github.com/nushell/nushell/) - An attractive structured shell
+* [starship](https://github.com/starship/starship) - The minimal, blazing-fast, and infinitely customizable prompt for any shell!
```

Comment:
starship is not shell but cross-shell prompt. So, I think `bash/PowerShell/fish` subsection should not have starthip.

Reply:
Sorry, I’m a bit confused about this category, I don’t know where it should be placed.

Reply:
Hmm. I have no idea about good original item to which starship is an alternative. Considering this repository is a curated list of replacements for existing software written in Rust, I think starship should not be added even though starship is a great project.

Reply:
respect your suggestion